unittests/application.loadPlaceFile.js

```diff
@@ -10,8 +10,10 @@ module.exports = test => {
     const doneFile = api.path.join(path, 'example.done');
     api.fs.unlinkSync(doneFile);
     app.loadPlaceFile(placeName, path, file, () => {
-      test.assert(api.fs.existsSync(doneFile));
-      test.end();
+      process.nextTick(() => {
+        test.assert(api.fs.existsSync(doneFile));
```

Comment:
This may make test flaky anyway.

Reply:
Maybe I should use `setTimeout` here, whdyt?